src/main/java/com/puppycrawl/tools/checkstyle/utils/TokenUtil.java

```diff
@@ -245,4 +245,16 @@ public static void forEachChild(DetailAST root, int type, Consumer<DetailAST> ac
         }
     }
 
+    /**
+     * Determines if two ASTs are on the same line.
+     *
+     * @param ast1   the first expression
+     * @param ast2   the second expression
+     *
+     * @return true if they are, false otherwise
```

Comment:
https://docs.oracle.com/javase/7/docs/api/java/util/List.html#isEmpty()
The return javadoc should be a complete sentence imo.

Reply:
changed